launchable/commands/record/commit.py

```diff
@@ -74,16 +74,38 @@ def commit(ctx, source: str, executable: bool, max_days: int, scrub_pii: bool, i
         )
         client.print_exception_and_recover(e)
 
+    cwd = os.path.abspath(source)
     try:
-        exec_jar(os.path.abspath(source), max_days, ctx.obj, is_collect_message)
+        is_shallow = subprocess.check_output(
+            ['git', 'rev-parse', '--is-shallow-repository'],
+            stderr=subprocess.DEVNULL,
+            cwd=cwd,
+            universal_newlines=True).strip()
+        if is_shallow == "true":
+            click.echo(click.style(
+                "Can't collect commit history from {} since it is the shallow repository. "
+                "Please use full clone or disable commit collection by --no-commit-collection option."
```

Comment:
nits: Some tenants doing a shallow clone, intentionally. So this message might not be necessary